.github/workflows/build.yml

```
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: ["18.12.0"]
+        node-version: ["21.5.0"]
```

```suggestion
        node-version: ["20.11.1"]
```
Prefer using LTS versions